File: ServidorDeCanciones/capaFachadaServices/DTOs/CancionAlmacenarDTOInput.go
package dtos

// Representa los datos necesarios para almacenar una nueva canción
type CancionAlmacenarDTOInput struct {
	Titulo        string `json:"titulo"`
	Artista       string `json:"artista"`
	Album         string `json:"album"`
	Anio          int    `json:"anio"`
	Duracion      string `json:"duracion"`
	Genero        string `json:"genero"`
	FechaRegistro string `json:"fecha_registro"`
}

File: ServidorDeEnvioCorreo/src/main/java/co/edu/unicauca/infoii/correo/DTOs/CancionAlmacenarDTOInput.java
package co.edu.unicauca.infoii.correo.DTOs;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class CancionAlmacenarDTOInput {	
    private String titulo;
    private String artista;
    private String genero;
    private String album;
    private int anio;
    private String duracion;
    private String fecha_registro;
    private String idioma;
}